eth/downloader/downloader_test.go

```diff
@@ -1613,12 +1699,17 @@ func TestRemoteHeaderRequestSpan(t *testing.T) {
 
 // Tests that peers below a pre-configured checkpoint block are prevented from
 // being fast-synced from, avoiding potential cheap eclipse attacks.
-func TestCheckpointEnforcement64Full(t *testing.T)  { testCheckpointEnforcement(t, 64, FullSync) }
-func TestCheckpointEnforcement64Fast(t *testing.T)  { testCheckpointEnforcement(t, 64, FastSync) }
+func TestCheckpointEnforcement64Full(t *testing.T) { testCheckpointEnforcement(t, 64, FullSync) }
+func TestCheckpointEnforcement64Fast(t *testing.T) { testCheckpointEnforcement(t, 64, FastSync) }
```

Comment:
It's a bit confusing to me. The `eth65` only adds the transaction retrievals, it did nothing at the syncing level. Shouldn't we also test the `lightSync` on eth64?

Reply:
Yeah, probably. I didn't add any new-old tests, just new-new tests